packages/hash/frontend/src/components/BlockLoader/BlockLoader.tsx

```diff
@@ -93,13 +102,16 @@ export const BlockLoader: VoidFunctionComponent<BlockLoaderProps> = ({
   const functions = {
     aggregateEntityTypes,
     aggregateEntities,
-    createLinks,
-    deleteLinks,
```

Comment:
This was previously alphabetical, now it is a mixture of alphabetical and grouped by target – can we do one or the other? e.g. if we want to group by target, `updateEntities` should be next to `aggregateEntities`. Personally I feel that alphabetical is easier to reason about, because you don't have to think about grouping, but not a strong opinion.

Reply:
Apologies I didn't notice the alphabetical ordering, have addressed this in https://github.com/hashintel/hash/pull/497/commits/d6608c89e90520c182058f8725f08a2fbe150a73